scrapy/downloadermiddlewares/stats.py

```diff
@@ -24,7 +43,7 @@ def process_request(self, request, spider):
     def process_response(self, request, response, spider):
         self.stats.inc_value('downloader/response_count', spider=spider)
         self.stats.inc_value(f'downloader/response_status_count/{response.status}', spider=spider)
-        reslen = len(response_httprepr(response))
+        reslen = len(response.body) + get_header_size(response.headers) + get_status_size(response.status) + 4
```

Comment:
It looks like this was the only usage of `response_httprepr`. May be deprecate the function? (i.e. log a deprecation warning when it is used, lookup usage of `ScrapyDeprecation` in code).

Also, maybe worth commenting that 4 stands for the 2 `\r\n` between those 3 sections.

Reply:
explanation (comment) added https://github.com/scrapy/scrapy/pull/4972/commits/915bd9cca933f3050250186b0479c9c71e475751
`response_httprepr` deprecated https://github.com/scrapy/scrapy/pull/4972/commits/f0a4d7857c282bf52829455d3300f3b28e366cdb